packages/@aws-cdk/aws-lambda/test/function.test.ts

```diff
@@ -684,6 +685,85 @@ describe('function', () => {
     })).toThrow(/deadLetterQueue defined but deadLetterQueueEnabled explicitly set to false/);
   });
 
+  test('default function with SNS DLQ when client provides Topic to be used as DLQ', () => {
+    const stack = new cdk.Stack();
+
+    const dlTopic = new sns.Topic(stack, 'DeadLetterTopic');
+
+    new lambda.Function(stack, 'MyLambda', {
+      code: new lambda.InlineCode('foo'),
+      handler: 'index.handler',
+      runtime: lambda.Runtime.NODEJS_10_X,
+      deadLetterTopic: dlTopic,
+    });
+
+    const tpl = Template.fromStack(stack);
```

Comment:
```suggestion
    const template = Template.fromStack(stack);
```

(we just do that everywhere else)

Reply:
Fair enough :D. I do `tpl` everywhere but that's your repo :D